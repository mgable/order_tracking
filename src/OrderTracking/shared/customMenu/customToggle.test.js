import React from 'react';
import ReactDOM from 'react-dom';
import { shallow, render } from 'enzyme';
import { CustomToggle } from './';

let props = {onClick: jest.fn()}

it('renders CustomToggle without crashing', () => {
    const wrapper = render(<CustomToggle props = { {...props} } />);
  	expect(wrapper).toMatchSnapshot();
});

