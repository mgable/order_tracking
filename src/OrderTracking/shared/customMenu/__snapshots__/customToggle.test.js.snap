// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CustomToggle without crashing 1`] = `
<button
  class="btn btn-primary"
  type="button"
/>
`;
